custom_components/eufy_security/coordinator.py

```diff
@@ -164,6 +164,8 @@ async def set_devices(self):
             _LOGGER.debug(
                 f"{DOMAIN} - connect - async_start_listening False - {self.config.__dict__}"
             )
+            # close down the websocket to fully retry the connection
+            await self.ws.ws.close()
```

Comment:
have you checked this against captcha condition? I was intentionally leaving the connection open as to enter captcha, add-on (web socket) should keep the internal state.

Reply:
I've not checked that path. I've never yet been prompted for a captcha, and I could see how closing the websocket while going through the captcha process. Adding a check for `self.captcha_config.required == False` may be prudent (if I understand the code correctly).